src/native/libs/System.Globalization.Native/CMakeLists.txt

```diff
@@ -84,7 +84,10 @@ endif()
 
 if (NOT CLR_CMAKE_TARGET_MACCATALYST AND NOT CLR_CMAKE_TARGET_IOS AND NOT CLR_CMAKE_TARGET_TVOS AND DEFINED CMAKE_ICU_DIR)
     include_directories(${CMAKE_ICU_DIR}/include)
-    link_libraries(${CMAKE_ICU_DIR}/lib/libicuuc.a ${CMAKE_ICU_DIR}/lib/libicui18n.a ${CMAKE_ICU_DIR}/lib/libicudata.a)
+    link_libraries(${CMAKE_ICU_DIR}/lib/libicuuc.a ${CMAKE_ICU_DIR}/lib/libicui18n.a)
+    if (NOT CLR_CMAKE_TARGET_BROWSER)
+        link_libraries(${CMAKE_ICU_DIR}/lib/libicudata.a)
```

Comment:
Do you want to introduce a "real" statically-linked-data S.N.Globalization at some point (`libicudata.a` is of course just a no-op stub)? I can port that one from downstream as well...